tools/rmlvo-to-keymap.c

```diff
@@ -64,6 +65,9 @@ usage(char **argv)
            "    Enable verbose debugging output\n"
            " --print-kccgst\n"
            "    Print the KcCGST names instead of the full keymap\n"
+           " --from-kccgst\n"
```

Comment:
The input is not necessarily KcCGST, it can be any xkb file (and I occasionally use it to get the compiled form of some xkb file). So maybe `--from-xkb`?

I understand it will make it less obvious that you can pipe ``--print-kccgst`` into ``--from-kccgst``. So I suggest changing ``--print-kccgst`` to only ``--kccgst`` with this help text:

> Print a keymap which only includes the KcCGST instead of the fully processed keymap

Reply:
just for clarification - what do you mean by "any xkb file"? AFAIU and with a short trial-and-error it needs to be a `xkb_keymap` which can be either the KcCGST only or the full keymap. Maybe `--from-xkb-keymap` is a better option then? 
I've changed it to `--from-xkb` for now though.

Reply:
By analogy to RMLVO I usually take KcCGST to mean just the 5 values (`keycodes = evdev+aliases(qwerty)` etc).

`--from-xkb-keymap` is also good though annoying to type.

Reply:
that's what bash/zsh completion is for ;)